spec/javascripts/components/story/story_item_spec.js

```diff
@@ -80,4 +80,38 @@ describe('<StoryItem />', () => {
       expect(children).toHaveProp('title', title);
     });
   });
+
+  describe('when the story is accepted', () => {  
+    let children;
```

Comment:
What do you think about creating a render function here that already returns the expanded story for you inside an object? Something like that:

```js
const render = (props) => {
  const wrapper = shallow(<StoryItem {...props} />;
  const expandedStory = wrapper.find(ExpandedStory) // I'd use a data-id here instead of that
  return { wrapper, expandedStory }; // notice the use _expandedStory_, not children, making it more descriptive
}

// and then in the tests you do:

const { expandedStory } = render({
  story: storyFactory({ ... })
});

// and then use `expandedStory` for the assertion 😄
```